File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
class Solution(object):
    def maximumCount(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        negativeCount , positiveCount = 0 , 0 
        for num in nums :
            if num  < 0 :
                negativeCount += 1 
            elif num > 0 :
                positiveCount += 1 
        return max(negativeCount , positiveCount)

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.c
int maximumCount(int* nums, int numsSize) {
    int negativeCount = 0  , positiveCount = 0 ; 
    for(int i = 0 ; i < numsSize ; i++)
    {
        if(nums[i] < 0 )
        {
            negativeCount++ ; 
        }
        else if (nums[i] >  0)
        {
            positiveCount++ ; 
        }
    }
    return (negativeCount > positiveCount) ? negativeCount : positiveCount;
}